spec/features/students/add_class_spec.rb

```diff
@@ -47,12 +47,23 @@
 
     end
 
-    context 'if it the code is not valid' do
+    context 'if the code is not valid' do
 
-      it 'displays an error message' do
+      it 'displays the standard error message' do
         page.find(".class-input").set('not-a-class')
         click_button('Join Your Class')
-        eventually {expect(page).to have_content("Invalid Classcode")}
+        eventually {expect(page).to have_content("Oops! Looks like that isn't a valid class code. Please try again.")}
+      end
+
+      it 'displays the archived class error message' do
```

Comment:
Lines 59 to 63 can be refactored into one line with `Classroom.create(name: 'Archived Class, visible: true....`